test/integration/targets/win_dns_record/tasks/tests-PTR.yml

```diff
@@ -0,0 +1,186 @@
+- name: 'TYPE=PTR - creation (check mode)'
+  win_dns_record: {zone: '{{ win_dns_record_revzone }}', name: 7, value: ansible-mirror.example.com, type: PTR}
+  register: cmd_result
+  check_mode: yes
+
+- name: 'TYPE=PTR - creation get results (check mode)'
+  win_command: powershell.exe "If (Get-DnsServerResourceRecord -Zone '{{ win_dns_record_revzone }}' -Name '7' -RRType PTR -Node -ErrorAction:Ignore) { 'exists' } else { 'absent' }"
```

Comment:
In order to make the tests work for meI have to substitute in this file all instances of `-Zone` by `-ZoneName` 
See https://github.com/johnboy2/ansible/pull/2